api/rpc/account/account.go

```diff
@@ -0,0 +1,307 @@
+package account
+
+import (
+	"fmt"
+	"net/mail"
+
+	context "golang.org/x/net/context"
+
+	google_protobuf1 "github.com/golang/protobuf/ptypes/empty"
```

Comment:
I know there are samples on the Internet using this convention because that's what protoc generates for `*.pb.go` file imports, ie, `google_protobuf1`, `google_protobuf2`, etc), but I don't think we need to follow that convention for human generated code. I'm fine with either of the following:

```
import (
        .  "github.com/golang/protobuf/ptypes/empty"    //->   Empty
        // or
        pb "github.com/golang/protobuf/ptypes/empty"    // -> pb.Empty
)

I'm fine with the dot import form simply because [empty.proto](https://github.com/google/protobuf/blob/master/src/google/protobuf/empty.proto) only exports the single Empty message anyway.